orders.go

```diff
@@ -9,6 +9,8 @@ type Orders []Order
 
 // Order is a customer's completed request to purchase one or more products from a shop.
 type Order struct {
+	// Attributes represents the custom order attributes for the order, if there are any.
+	Attributes OrderAttributes
```

Comment:
Is it not `NoteAttributes` as it seems to be `note_attributes` in Shopify?

Reply:
Yeah I guess it should be called that.